lib/tasks/db.js

```diff
@@ -53,13 +104,17 @@ module.exports = {
         migrator.pending()
         .then(function (seeders) {
           if (seeders.length === 0) {
-            console.log('No seeders were found.');
+            console.log('No seeders found.');
             process.exit(0);
           }
-        }).then(function () {
-          return migrator.up();
+
+          return migrator.up(_.chain(seeders).pluck('file').value());
```

Comment:
Can we streamline the up/down calls a bit and either pass an object with `migrations` or the array?